lte/gateway/c/li_agent/CMakeLists.txt

```diff
@@ -93,12 +93,14 @@ link_directories(
   ${MAGMA_LIB_DIR}/service_registry)
 
 add_library(LI_AGENT
+    Utilities.h
```

Comment:
Unrelated to this PR but we should file an issue to sanitize all the proto generation.
```
set(SMGR_LTE_CPP_PROTOS mconfig/mconfigs)generate_cpp_protos("${SMGR_LTE_CPP_PROTOS}" "${PROTO_SRCS}"  "${PROTO_HDRS}" ${LTE_PROTO_DIR} ${LTE_CPP_OUT_DIR})set(SMGR_CWF_CPP_PROTOS accounting context)generate_cwf_cpp_protos("${SMGR_CWF_CPP_PROTOS}" "${PROTO_SRCS}"  "${PROTO_HDRS}" ${CWF_PROTO_DIR} ${CWF_CPP_OUT_DIR})set(SMGR_ORC8R_GRPC_PROTOS directoryd)generate_grpc_protos("${SMGR_ORC8R_GRPC_PROTOS}" "${PROTO_SRCS}"  "${PROTO_HDRS}" ${ORC8R_PROTO_DIR} ${ORC8R_CPP_OUT_DIR})message("Proto_srcs are ${PROTO_SRCS}")
```
we shouldn't have to generate any cwf / accounting protos

Reply:
https://github.com/magma/magma/pull/7359

Reply:
thanks @themarwhal